### PR TITLE
JS-WeatherApp

### PR DESCRIPTION
**Pull request by Jhonatanssal**
### This app uses technologies:
- JavaScript
- OpenWeather API
- Bootstrap
- Webpack

Hope it accomplishes all of the requirements.
Happy coding. 👍🏼

### CODE REVIEW

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi there @jssarrazolaa :hand: 
Awesome job on this milestone so far :clap: :+1: 
But we need to make some changes before moving on.

- [x] As is said in the instructions, the app should have a toggle control to determine if the result will be shown in Celsius or Fahrenheit.
Screenshot from 2020-11-09 20-10-23

- [x] Your app should have an error handling for the API call. You can create a hidden message inside a container that's only visible when the API call returned an error. Can be something like Location not found. 
Screenshot from 2020-11-09 20-22-52

- [x] The .eslintrc.json file is different from the one provided by Microverse. We need to use this version [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json).

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

File: .eslintrc.json [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off",
+    "max-len": "off"
```
Comment (bellom) [inline .eslintrc.json]: As mentioned by the previous reviewer, we need to use the standard `eslintrc.json` from microverse.

we have just one line changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/javascript)

image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

File: src/dom.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          buttonC.style.display = 'block';
+        });
+
+        buttonC.addEventListener('click', () => {
+          temp.textContent = `${item.main.temp} C°`;
+          buttonF.style.display = 'block';
+          buttonC.style.display = 'none';
+        });
+      })
+      .catch(() => {
```
Comment (bellom) [inline src/dom.js]: Let's separate the DOM from application logic, you can create another file for DOM.

In your case, we have DOM task inside the API logic.

image

https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635172/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @jssarrazolaa  :clap: 

**Great work making `almost` all the necessary corrections from the previous review.**

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by Torres-ssf — APPROVED** [review]
### PR Approved :clap: :+1: 
Hi there @jssarrazolaa :hand: 
Great job implementing the requested changes :smile: 
Good luck with the next project

Happy Coding :computer: 
**Sergio Torres**

File: src/dom.js (hunk 2 above)
Comment (Jhonatanssal) [reply src/dom.js]: Thanks for the review! 👍🏼

**Pull request merged**